### PR TITLE
Update UG for Expected Output of tasks

### PR DESCRIPTION
**Pull request by maj0-0**
lmk if anything else i need to update!

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -149,7 +152,7 @@ Exits the program.
 
 Adds a student to the system.
 
-#### Format: `add n/NAME m/MAJOR y/YEAR e/EMAIL d/DESCRIPTION [t/TUTORIALS] [sm/SOCIALMEDIA] nt/NATIONALITY g/GENDER`
+#### Format: `add n/NAME m/MAJOR y/YEAR e/EMAIL d/DESCRIPTION [t/TUTORIALS]… [sm/SOCIALMEDIA]… nt/NATIONALITY g/GENDER`
```
Comment (wnchan) [inline docs/UserGuide.md:155]: need change to this in add command file too

**Review by wnchan — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -59,17 +59,18 @@ StudentConnect is a solution for hassle-free team formation for students to brow
...
-    * `clear` : Deletes all data from the system (i.e. students, groups, and tasks).
-
-    * `exit` : Exits the app.
-
-    * `create` : Creates a new group.
+    * `list` : 
+        * Lists all students in the StudentConnect system.
+    * `add n/John Doe m/Computer Science y/2 e/johnd@u.nus.edu d/I love programming in my free time t/02 t/17 t/20 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/m` : 
+        * Adds a student named `John Doe` and their details to the StudentConnect system.
+    * `delete [email]` :
```
Comment (PearlynnT) [inline docs/UserGuide.md:66]: should we change this to `delete EMAIL` since this is the given format below

Comment (maj0-0) [reply docs/UserGuide.md]: okay!

File: docs/UserGuide.md (hunk 1 above)
Comment (maj0-0) [reply docs/UserGuide.md:155]: i think it's alr there!

**Review by neyapraveen — COMMENTED** [review]
LGTM!

**Pull request merged**